src/OpenAL/OpenToolkit.OpenAL/ALC/ALCEnums.cs

```diff
@@ -0,0 +1,182 @@
+        AllAttributes = 0x1003,
+
+        /// <summary>The number of capture samples available. NULL is an invalid device.</summary>
+        CaptureSamples = 0x312,
+
+        /// <summary>(EFX Extension) This property can be used by the application to retrieve the Major version number of the Effects Extension supported by this OpenAL implementation. As this is a Context property is should be retrieved using alcGetIntegerv.</summary>
+        EfxMajorVersion = 0x20001,
+
+        /// <summary>(EFX Extension) This property can be used by the application to retrieve the Minor version number of the Effects Extension supported by this OpenAL implementation. As this is a Context property is should be retrieved using alcGetIntegerv.</summary>
+        EfxMinorVersion = 0x20002,
+
+        /// <summary>(EFX Extension) This Context property can be passed to OpenAL during Context creation (alcCreateContext) to request a maximum number of Auxiliary Sends desired on each Source. It is not guaranteed that the desired number of sends will be available, so an application should query this property after creating the context using alcGetIntergerv. Default: 2</summary>
+        EfxMaxAuxiliarySends = 0x20003,
+    }
+
+    /// <summary>
+    /// Defines available parameters for <see cref="ALC.GetString(ALDevice, GetEnumerationString)" />.
+    /// </summary>
+    public enum GetEnumerationString
+    {
+        /// <summary>
+        /// Gets the specifier for the default device. ALC_ENUMERATION_EXT
+        /// </summary>
+        DefaultDeviceSpecifier = 0x1004,
+
+        /// <summary>
+        /// Gets a specific output device's specifier.
+        /// Can also be used without a device to get a list of all available output devices, see <see cref="GetEnumerationStringList.DeviceSpecifier"/>. ALC_ENUMERATION_EXT
+        /// </summary>
+        DeviceSpecifier = 0x1005,
+
+        /// <summary>
+        /// Gets the specifier for the default capture device. ALC_ENUMERATION_EXT
+        /// </summary>
+        DefaultCaptureDeviceSpecifier = 0x311,
+
+        /// <summary>
+        /// Gets a specific capture device's specifier.
+        /// Can also be used without a device to get a list of all available capture devices, see <see cref="GetEnumerationStringList.CaptureDeviceSpecifier"/>. ALC_ENUMERATION_EXT
+        /// </summary>
+        CaptureDeviceSpecifier = 0x310,
+    }
+
+    /// <summary>
+    /// Defines available parameters for <see cref="ALC.GetStringList(ALDevice, GetEnumerationStringList)" />.
+    /// </summary>
+    public enum GetEnumerationStringList
+    {
+        /// <summary>
+        /// Gets the specifier strings for all available output devices.
+        /// Can also be used to get the specifier for a specific device, see <see cref="GetEnumerationString.DeviceSpecifier"/>. ALC_ENUMERATION_EXT
+        /// </summary>
+        DeviceSpecifier = 0x1005,
```

Comment:
These enums used to be in plural, as in `DeviceSpecifiers` but in the AL spec they are called `ALC_DEVICE_SPECIFIER` without the `s`. But due to this enum existing in the about `GetEnumerationString` enum this could become confusing. The one in `GetEnumerationString` is meant to be used to get the specifier for a device while this is mean to be used to get a list of available specifiers. I'm not sure what is the best decision here.

Reply:
Worst case, people upgrading have to remove the plural s from their code, right?

Reply:
Yeah I think this is the way to go. (Following the naming of the openal standard)